i18n/en.json

```diff
@@ -138,11 +136,12 @@
   "I18N_TOPNAV_CONTACT_US": "Contact Us",
   "I18N_TOPNAV_LIBRARY": "Library",
   "I18N_TOPNAV_FORUM": "Forum",
+  "I18N_ACTION_BROWSE_EXPLORATIONS": "Browse our Explorations",
+  "I18N_ACTION_CREATE_EXPLORATION": "Create an Exploration",
   "I18N_CREATE_EXPLORATION_CREATE": "Create",
   "I18N_CREATE_EXPLORATION_UPLOAD": "Upload",
   "I18N_CREATE_ACTIVITY_TITLE": "Create an Activity",
   "I18N_CREATE_ACTIVITY_QUESTION": "What do you want to create?",
-  "I18N_CREATE_EXPLORATION_TITLE": "Create an Exploration",
```

Comment:
You need to be really careful when deleting stuff. Did you use grep to see if this is used anywhere else?